### PR TITLE
feature - register screen

### PR DESCRIPTION
**Pull request by at-phatnguyen**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Register/RegisterViewController.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        let backBarButton = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        navigationItem.backBarButtonItem = backBarButton
+        let rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(register))
+        navigationItem.rightBarButtonItem = rightBarButtonItem
+
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Register/RegisterViewController.swift]: We should delete unused code

File: MyApp/Define/String.swift [hunk 2]
```diff
@@ -27,5 +27,8 @@ extension App {
         static let picture4 = "4"
         static let home = "Home"
         static let profile = "Profile"
+        static let register = "Register"
+        static let back = "Back"
+        static let forgotPass = "Forgot PassWord"
```
Comment (at-truongnguyen) [inline MyApp/Define/String.swift]: `Password` not `PassWord`

File: MyApp.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -12,6 +12,10 @@
 		5221D51C1F29912A008940CA /* LeftViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5221D51A1F29912A008940CA /* LeftViewController.swift */; };
 		5221D51D1F29912A008940CA /* LeftViewController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 5221D51B1F29912A008940CA /* LeftViewController.xib */; };
 		5221D51F1F2A4C80008940CA /* LoginViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5221D51E1F2A4C80008940CA /* LoginViewModel.swift */; };
+		5230877F1F2EF1B4004BA6BC /* RegisterViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5230877D1F2EF1B4004BA6BC /* RegisterViewController.swift */; };
+		523087801F2EF1B4004BA6BC /* RegisterViewController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 5230877E1F2EF1B4004BA6BC /* RegisterViewController.xib */; };
+		524C92B91F2F0F5E003FCDCB /* ForgotPassWordViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 524C92B71F2F0F5E003FCDCB /* ForgotPassWordViewController.swift */; };
+		524C92BA1F2F0F5E003FCDCB /* ForgotPassWordViewController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 524C92B81F2F0F5E003FCDCB /* ForgotPassWordViewController.xib */; };
```
Comment (at-truongnguyen) [inline MyApp.xcodeproj/project.pbxproj]: `Password` not `PassWord`

**Pull request merged**